Source/ASDisplayNode.mm

```diff
@@ -3271,6 +3271,18 @@ - (UIEdgeInsets)hitTestSlop
   return _hitTestSlop;
 }
 
+- (BOOL)isRTL {
+  if (AS_AVAILABLE_IOS_TVOS(10, 10)) {
+    return _primitiveTraitCollection.layoutDirection == UITraitEnvironmentLayoutDirectionRightToLeft;
+  } else {
+    return [UIView userInterfaceLayoutDirectionForSemanticContentAttribute:_view.semanticContentAttribute] == UIUserInterfaceLayoutDirectionRightToLeft;
```

Comment:
The `_view` ivar will be nil if the node is not yet loaded so you need to nil check it before using.